ui/src/App.test.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
```

Comment:
Intriguing  :)
I've never imagined how render of an element could be tested

Reply:
All is simulated via a JS engine that acts nearly as actual Web browser render engine.